libs/core/src/scss/base/_variables.scss

```
@@ -74,6 +74,7 @@ $z-layers: (
   default: 1,
   segment-badge: 2,
   avatar-badge: 2,
+  sticky-content: 169,
```

It seems like a `z-index` of 2 would be sufficient :) I tested it out in different scenarios (including the reorder list) and the layer is still on top.
```suggestion
  sticky-content: 2,
```